packages/server/src/routes/index/users.ts

```diff
@@ -246,6 +248,9 @@ router.post('users', async (path: SubPath, ctx: AppContext) => {
 			// that user, except the current one (otherwise they would be
 			// logged out).
 			if (userToSave.password) await models.session().deleteByUserId(userToSave.id, contextSessionId(ctx));
+		} else if (fields.send_account_confirmation_email) {
+			await models.user().save({ id: user.id, must_set_password: 1 });
```

Comment:
Any reason why you set `must_set_password` here?

Reply:
No, just a copy-and-paste badly done.

Reply:
No, I've copied and pasted it as it was in the admin/users.ts. Maybe I thought the must_set_password was required. But, well, I removed it.